docker/webui/Dockerfile

```diff
@@ -1,10 +1,12 @@
+#!BuildTag: openqa_webui
 FROM opensuse/leap:15.2
 LABEL maintainer Jan Sedlak <jsedlak@redhat.com>, Josef Skladanka <jskladan@redhat.com>, wnereiz <wnereiz@eienteiland.org>, Sergio Lindo Mansilla <slindomansilla@suse.com>
 LABEL version="0.3"
 
-RUN zypper ar -f obs://devel:openQA/openSUSE_Leap_15.2 openQA && \
-    zypper ar -f obs://devel:openQA:Leap:15.2/openSUSE_Leap_15.2 openQA-perl-modules && \
-    zypper --gpg-auto-import-keys ref && \
+#RUN zypper ar -f -G http://download.opensuse.org/repositories/devel:/openQA:/Leap:/15.2/openSUSE_Leap_15.2 openQA && \
```

Comment:
not sure if the line continuation marker at EOL will not interfer despite the disabled line